rld/lib/ErrorCode.cpp

```diff
@@ -22,6 +22,6 @@ std::string rld::ErrorCategory::message(int error) const {
 }
 
 std::error_category const &rld::ErrorCategory::get_error_category() {
-  static ErrorCategory const cat;
+  static ErrorCategory const cat{};
```

Comment:
Curious about this explicit initialization. The class has a default ctor which should render this unnecessary.